packages/eslint-plugin/src/rules/jsx-pragma/__tests__/rule.test.tsx

```diff
@@ -148,5 +148,95 @@ import { css } from '@compiled/react';
         },
       ],
     },
+    {
+      code: `
+        import React from 'react';
+        import { css } from '@compiled/react';
+        <div css={css({ display: 'block' })} />
+      `,
+      output: `
+        /** @jsxImportSource @compiled/react */
+
+        import { css } from '@compiled/react';
+        <div css={css({ display: 'block' })} />
+      `,
+      errors: [
+        {
+          messageId: 'missingPragma',
+        },
+      ],
+    },
+    {
+      code: `
+        import React from 'react';
+        import { css } from '@compiled/react';
+
+        React.useState();
+
+        <div css={css({ display: 'block' })} />
+      `,
+      output: `
+        /** @jsxImportSource @compiled/react */
+import React from 'react';
+        import { css } from '@compiled/react';
+
+        React.useState();
+
+        <div css={css({ display: 'block' })} />
+      `,
+      errors: [
+        {
+          messageId: 'missingPragma',
+        },
+      ],
+    },
+    {
+      code: `
+        import React, { useState } from 'react';
+        import { css } from '@compiled/react';
+
+        useState();
+
+        <div css={css({ display: 'block' })} />
+      `,
+      output: `
+        /** @jsxImportSource @compiled/react */
+import { useState } from 'react';
+        import { css } from '@compiled/react';
+
+        useState();
+
+        <div css={css({ display: 'block' })} />
+      `,
+      errors: [
+        {
+          messageId: 'missingPragma',
+        },
+      ],
+    },
+    {
+      code: `
+        import React,{useState } from 'react';
```

Comment:
What about some test cases for `import { useState } from 'react';` and `import * as React from 'react';`?

Reply:
Will add! Nice eye.